### PR TITLE
Allow any number of arguments that are strings

### PR DESCRIPTION
**Pull request by jmclean-cnexus**
Update Typings for `FilterFunction` to reflect usage on https://squirrelly.js.org/docs/api/filter-api, allowing multiple input args.

### CODE REVIEW

**Comment by nebrelbug** [comment]
@jmclean-cnexus thanks for the PR!

File: src/containers.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export type HelperFunction = (
   config: SqrlConfig
 ) => string | Promise<string>
 
-export type FilterFunction = (str: string) => any | Promise<any>
+export type FilterFunction = (...args: string[]) => any | Promise<any>
```
Comment (nebrelbug) [inline src/containers.ts:28]: Will you change the signature to something like `(val: any, ...filterParams: any[])`?

A filter function should probably always take at least the first parameter. Also, filters can actually accept more than just strings. For example, `{{ it.arr | join(",") | truncate(3) }}` would be completely valid syntax.

Thanks again!

**Review by nebrelbug — CHANGES_REQUESTED** [review]
Looks good! If you could just update based on my comment, that would be perfect.

File: src/containers.ts (hunk 1 above)
Comment (jmclean-cnexus) [reply src/containers.ts:28]: I will make that change now! was just following with

**Comment by jmclean-cnexus** [comment]
@nebrelbug anything else i can do to the PR to get it to merge. BTW...thanks for such an awesome module; I think ill be using/recommending it for many of my future solutions.

**Comment by nebrelbug** [comment]
@jmclean-cnexus this looks great! I'll merge now :tada:.

> BTW...thanks for such an awesome module; I think ill be using/recommending it for many of my future solutions.

Thank you! I'm glad you enjoy it!

By the way, is it alright if I add you to the README as a contributor?

**Comment by jmclean-cnexus** [comment]
I would be proud to be on the readme, and will hopefully provide more quality PRs in the future!

**Comment by nebrelbug** [comment]
@jmclean-cnexus awesome! I hope you can!

@all-contributors please add @jmclean-cnexus for his coding contributions :tada:

**Pull request merged**